JavaScript/Active_pull_request_commentors/script.js

```diff
@@ -0,0 +1,79 @@
+const fetch = require("node-fetch");
+const createCsvWriter = require("csv-writer").createObjectCsvWriter;
+require('dotenv').config();
+
+let input = process.argv.slice(2);
+let OWNER = input[0]; 
+let REPO_NAME = input[1]; 
+let NO_OF_REQUEST = input[2]; 
+let obj;
+const query = `query {
+    repository(owner: "${OWNER}", name: "${REPO_NAME}") {
+      url
+      pullRequests(first:${NO_OF_REQUEST}, states:OPEN) {
+        nodes {
+          number
+          url
+          state
+          author {
+            login
+            url
+          }
+          comments(first:100) {
+            nodes {
+              author {
+                login
+              }
+              bodyText
+            }
+          }
+        }
+      }
+    }
+  }
+  `;
+
+fetch("https://api.github.com/graphql", {
```

Comment:
Fetch isn't the best practice in JS.
Modern ES6+ practices specify to use `async await` functions to perform fetch calls.
You can abstract this piece out into an async function and `await` on the https://api.github.com/graphql link and then perform try catch on that.

[This](https://javascript.info/async-await) will help you a lot